api/metadata.proto

```diff
@@ -71,12 +71,16 @@ message RxMetadata {
   uint32 hopping_width = 19;
   // Frequency drift in Hz between start and end of an LR-FHSS packet (signed).
   int32 frequency_drift = 20;
+  // GPS sourced timestamp at the end of the transmission.
+  google.protobuf.Timestamp gps_time = 21;
+  // Time when the service received the message.
+  google.protobuf.Timestamp received_at = 22;
```

Comment:
I would add slightly more details to the comments here. We now are going to have 3 timestamps and that will be confusing in the absence of the explanation.

I would add a comment to `time` saying `Timestamp at the end of the transmission, provided by the gateway. The accuracy is undefined.`

For `gps_time` I would mention `Timestamp at the end of the transmission, provided by the gateway. Guaranteed to be based on a GPS PPS signal, with an accuracy of 1 millisecond.`.

For `received_at` I would mention `Timestamp at which the Gateway Server has received the message.`